prow/config/jobs/istio.yaml

```diff
@@ -124,6 +124,20 @@ jobs:
       - name: TEST_SELECT
         value: "-multicluster"
 
+  - name: integ-pilot-multicluster-tests
+    type: postsubmit
+    command:
+      - entrypoint
+      - prow/integ-suite-kind.sh
+      - --topology
+      - MULTICLUSTER
+      - test.integration.pilot.kube
+    requirements: [kind]
+    modifiers: [optional, hidden, skipped]
+    env:
+      - name: TEST_SELECT
+        value: "-multicluster"
```

Comment:
is this right? I guess it is since these tests don't have the MC label?

Reply:
Right

Reply:
Is `type: postsubmit` going to prevent me from using `/test` to trigger? Not sure if I'm just impatient or if that's why `/test integ-pilot-multicluster-tests_istio_postsubmit` isn't working.

Reply:
ah yeah, sorry. It actually makes sense as well; we don't want someone to run `/test release-istio_postsubmit` on an unapproved pr for example